setup.py

```diff
@@ -0,0 +1,53 @@
+from setuptools import find_packages, setup
+
+try:
+    import pypandoc
+    long_description = pypandoc.convert('README.md', 'rst')
```

Comment:
Is this conversion to RST still necessary now that [PyPI properly supports Markdown READMEs](https://dustingram.com/articles/2018/03/16/markdown-descriptions-on-pypi)?

Reply:
Probably not. I'd like to do some experimentation though, considering that the Markdown support didn't work with the latest release of wagtail-flags.